element-hq/element-x-android
# Implement month separator for the Gallery, and improve date rendering.

## Content


Implement month separator for the Gallery.
Improve day separator rendering in the timeline.
Use Today, Yesterday, and the name of the day if less than 7 days and do not render the year for the current year. Improve date format for the media viewer.
Rework how date and time are computed.
ActionListView: Time can take more space, so update the layout.

## Motivation and context


Part of the work on the media gallery + Closes #2238

## Screenshots / GIFs



## Tests



- Run the app in your current locale and check that all the date across the application look good.
- Change the language of the phone / of the app and observe that the app uses the new languages (especially to render date and time).

## Tested devices

- OS version(s):

## Checklist
